### PR TITLE
#187300191 Google login integration

### PR DESCRIPTION
**Pull request by sabatohura**
#### What does this PR do?

- Implement Google authentication on frontend

#### Description of Task to be completed?

- Google Login
- Generating user token
- Store user token in session storage

#### How should this be manually tested?
- App login with Google
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2698211/stories/187300191

### CODE REVIEW

**Review by tchamianest — APPROVED** [review]

**Review by denislohan — CHANGES_REQUESTED** [review]
PR title is wrong

**Comment by sabatohura** [comment]
> PR title is wrong

let re-work on it, Thanks

**Review by kayigmb — APPROVED** [review]

**Comment by sabatohura** [comment]
> PR title is wrong

fixed

**Review by Eli250 — CHANGES_REQUESTED** [review]

File: .env [hunk 1]
```diff
@@ -1 +1,4 @@
-NEXT_PUBLIC_URL=https://champs-ec-be.onrender.com/api
```
Comment (Eli250) [inline .env:1]: This file should not be pushed

**Review by Eli250 — CHANGES_REQUESTED** [review]

**Review by Eli250 — APPROVED** [review]

**Pull request merged**